hooks/basic/collector.py

```diff
@@ -250,11 +250,12 @@ def _resolve_work_path_template(self, properties, path):
         return work_path_template
 
 
-    def _get_item_context_from_path(self, parent_item, properties, path):
+    def _get_item_context_from_path(self, parent_item, properties, path, extra_entities=list()):
         """Updates the context of the item from the work_path_template/template, if needed.
 
         :param properties: properties of the item.
         :param path: path to build the context from.
+        :param extra_entities: list of entities to add to the context that don't already exist in it.
```

Comment:
Please rename this `default_entities` and the description should read: "a list of default entities to use during the creation of the :class:`sgtk.Context` if not found in the path"